docs/api/manager-api/getting-started.md.blade.php

```diff
@@ -0,0 +1,110 @@
+---
+title: Getting Started
+---
+
+# Getting Started
+
+The Manager API allows you to gather common node information, access logs, control running processes and handle ledger snapshots on demand.
+Requests should contain valid JSON RPC 2.0 data which are transformed over HTTP / HTTPS protocol using GET method.
+
+<x-alert type="warning">
+All HTTP requests have to be sent with the `Content-Type: application/vnd.api+json` header. If the header is not present, it will result in malformed responses or request rejections.
+</x-alert>
+
+## Installation
+
+The Manager API comes by default with an installation of core. In case you don't have it installed, you can add it manually by running `yarn global add @arkecosystem/core-manager`.
+
+## Alias
+
+`manager`
+
+## Configuration
+
+The Manager API comes with default configuration. Additionally you can customize API specific settings in .env or app.json file.
+
+<x-alert type="warning">
+We strongly recommend setting basic authentication using Argon2Id hashing method with custom secret and using HTTPS access only.
+</x-alert>
+
+### Argon2Id Authentication
+
+It is recommend to make configuration changes for defining user access and IP whitelisting within your app.json file.
+
+```javascript
+{
+    "package": "@arkecosystem/core-manager",
+    "options": {
+        "plugins": {
+            "whitelist": ["127.0.0.1"],
+            "basicAuthentication": {
+                "enabled": true,
+                "secret": "secret",
+                "users": [
+                     {
+                         "username": "username",
+                         "password": "$argon2id$v=19$m=4096,t=3,p=1$NiGA5Cy5vFWTxhBaZMG/3Q$TwEFlzTuIB0fDy+qozEas+GzEiBcLRkm5F+/ClVRCDY"
+                     }
+                ]
+            }
+        }
+    }
+}
+```
+
+The `whitelist` property is an `Array` consisting of IP addresses that you allow to make connections to the manager API.
+By default, only local access to the webhook API is allowed. This means that if you want to expose your webhook API to the outside, you'll need to explicitely add the IP addresses that you will use to this list \(recommended approach\).
+It is also possible to use wildcards to indicate a range of IPs \(e.g. `"12.34.56.*"`\) or even to allow everyone \(e.g. `"*"`\) \(not recommended\).
+
+### Token authentication
+
+Alternatively instead basic authentication, token authentication can be enabled (not recommend).
```

Comment:
Alternatively, instead of basic authentication token authentication can be enabled (not recommended).